_episodes/17.md

```diff
@@ -0,0 +1,416 @@
+
+If you want to show us some 💕, please [give us a ⭐ on GitHub](/star).
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/episode/17/trino-resurface.png"/><br/>
+Commander Bun Bun is diving deep to find anomalies!
+</p>
+
+## Resurface links
+
+ * [Resurface Site](https://resurface.io/)
+ * [Resurface Github](https://github.com/resurfaceio)
+ * [Resurface Slack](https://resurface.io/slack)
+
+## Guests
+ * Rob Dickinson, Co-founder and CEO of [Resurface Labs](https://resurface.io/)
+ ([@robfromboulder](https://twitter.com/robfromboulder))
+ * Martin Traverso, creator of Trino/Presto, and CTO at 
+ [Starburst](https://starburst.io) ([@mtraverso](https://twitter.com/mtraverso))
+
+## Concept of the week: Resurface and the Resurface connector
+
+### What is Resurface?
+Resurface is an API system of record, which is a fancy way of saying that 
+Resurface is a purpose-built database for API requests and responses. Like a 
+weblog or access log, but on steroids because Resurface runs on Trino.
+
+Why do you need a system of record for your APIs? Because otherwise you’re 
+guessing about how your APIs are used and attacked, and guessing doesn’t feel 
+good. Resurface helps your DevOps and security teams instantly find API 
+failures, slowdowns, and attacks -- easily, responsibly, and at scale.
+
+### How Resurface Differs from Logs & Metrics
+You probably use system monitoring tools, which tell you about what’s happening 
+on your systems. What code is running, what code is slow, and what error codes 
+are returned. That’s all great — but it still leaves a big gap between the 
+system-level events you can see, and what your API consumers actually see.
+
+Resurface helps you fill this gap with your own API system of record. Now your 
+customers, your DevOps team, and your security team all have the same view of 
+every transaction, because there is a record of the requests and responses.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/17/resurface-tcb1.png"/><br/>
+</p>
+
+The other obvious way to compare Resurface against other tools is to look at the
+data model. System monitoring gives you time-series metrics, or timestamped log
+messages with a severity and detail string. Resurface gives you all the request
+and response data fields, including headers and payloads, in a schema where all
+of those fields are discrete and searchable. Plus we add a bunch of helpful
+virtual and computed columns.
+
+<p align="center">
+<img align="center" width="75%" height="100%" src="/assets/episode/17/resurface-tcb2.png"/><br/>
+</p>
+
+### The Indexing Problem
+
+So we have this very descriptive data model, but there’s a problem here -- how
+to partition and index this data for efficient searching. Partitioning based on
+time is the obvious starting point, but within a time range, what then? Index
+everything?
+
+Most databases work best when a subset of the columns are constrained at once --
```

Comment:
Change the whole post to not use "we" since this is TCB and not the Resurface website.

Reply:
Or alternatively add some sort of intro that clarifies this is what Resurface says

Reply:
I went with the first one